src/OpenTelemetry.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -109,9 +105,11 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
     ICorProfilerInfo6* info6;
     hr = cor_profiler_info_unknown->QueryInterface(__uuidof(ICorProfilerInfo6), (void**)&info6);
 
-    if (SUCCEEDED(hr)) {
-        Logger::Debug("Interface ICorProfilerInfo6 found.");
-        is_net46_or_greater = true;
+    // Fail if net46+ not detected
+    // TODO: Update this to ICorProfilerInfo7 which corresponds to net461 and make CorProfilerBase.info_ have type ICorProfilerInfo7*
+    if (FAILED(hr)) {
+        Logger::Warn("DATADOG TRACER DIAGNOSTICS - Failed to attach profiler: interface ICorProfilerInfo6 not found.");
```

Comment:
We should probably change this to something like the following to be consistent with the other logging messages.
```suggestion
        Logger::Warn("OpenTelemetry TRACER DIAGNOSTICS - Failed to attach profiler: interface ICorProfilerInfo6 not found.");
```

Reply:
Fixed in 6862af3feb8e502d2247047e6db8a0ef54d4a6a3 . This should be the only log message that accidentally got introduced with Datadog in the log